CALENDAR_TRIGGER_README.md

```diff
@@ -0,0 +1,145 @@
+# Google Calendar Event Trigger
```

Comment:
Could you rather put this in the PR description directly and remove this file please? 🙏

Reply:
@Shadterra You can also put some (not all this doc) of the most important infos in the "living doc" (`@Schema` annotations at the class level and properties attributes level).